tests/tests.spec.js

```diff
@@ -0,0 +1,69 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage');
+const { ProductPage } = require('../pages/productPage');
+const { CartPage } = require('../pages/cartPage');
+const { standardUser } = require('../test_data/users.json');
+const { CheckoutPage } = require('../pages/checkoutPage');
+const { ConfirmationPage } = require('../pages/confirmationPage');
+const { CompletionPage } = require('../pages/completionPage');
+
+const sauceLabsBikeLight = "Sauce Labs Bike Light";
+const sauceLabsBackpack = "Sauce Labs Backpack";
+const sauceLabsFleeceJacket = "Sauce Labs Fleece Jacket";
```

Comment:
I would put those information in `data_test` folder in `products.json` file, similar to `users.json` solution.